backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Extensions/Parsing.cs

```diff
@@ -4,7 +4,7 @@ public static class Parsing
 {
     public static int? ParseNullableInt(this string value)
     {
-        if (string.IsNullOrEmpty(value))
+        if (string.IsNullOrEmpty(value) || value.Equals("null"))
```

Comment:
Am I correct in understanding that this line fixed empty constraint?

Reply:
just in case

we deleted `"null"` from QueryResult